apps/rush-lib/src/schemas/rush.schema.json

```diff
@@ -65,6 +65,11 @@
           "description": "Configures the strategy used to select versions during installation.  This feature requires PNPM version 3.1 or newer.  It corresponds to the \"--resolution-strategy\" command-line option for PNPM.  Possible values are \"fast\" and \"fewer-dependencies\".  PNPM's default is \"fast\", but this may be incompatible with certain packages, for example the \"@types\" packages from DefinitelyTyped.  Rush's default is \"fewer-dependencies\", which causes PNPM to avoid installing a newer version if an already installed version can be reused; this is more similar to NPM's algorithm.",
           "type": "string",
           "enum": ["fewer-dependencies", "fast"]
+        },
+        "incrementalBuildDependencyGranularity": {
+          "description": "This is the mode for tracking external dependencies to detect if a project needs to be rebuilt during an incremental build.",
```

Comment:
Defines the granularity of tracking external dependencies to detect if a project needs to be rebuilt during an incremental build. 
or something along those lines ?